### PR TITLE
Remove horizontal scrollbar

### PR DESCRIPTION
**Pull request by ifdattic**

### CODE REVIEW

File: quick_intro_pt1.rst [hunk 1]
```diff
@@ -634,14 +643,15 @@ and ``Basket`` class looking like this:
         public function addProduct($product)
         {
             $this->products[] = $product;
-            $this->productsPrice += $this->shelf->getProductPrice($product);
+            $this->productsPrice += $this->shelf
+                ->getProductPrice($product);
```
Comment (stof) [inline quick_intro_pt1.rst]: I'm not a fan of this formatting. Having the method call on the next line is less readable IMO.

Comment (ifdattic) [inline quick_intro_pt1.rst]: I'm not a big fan of this either. Which one should I do:
1. Put everything after `=` on new line with indentation
2. Leave it all on the same line (like it was before)

Comment (stof) [inline quick_intro_pt1.rst]: I would keep it as it was for this case

**Comment by stof** [comment]
thanks

**Comment by ifdattic** [comment]
Glad to help

**Pull request merged**